src/resources/Microsoft.Authorization/policyDefinitions/Deny-AA-child-resources.json

```diff
@@ -11,7 +11,12 @@
     "metadata": {
       "version": "1.0.0",
       "category": "Automation",
-      "source": "https://github.com/Azure/Enterprise-Scale/"
+      "source": "https://github.com/Azure/Enterprise-Scale/",
+      "alzCloudEnvironments": [
```

Comment:
Should be only `AzureCloud` and `AzureUSGovernment` (possibly works on `AzureChinaCloud`, but not yet tested)